homeassistant/components/habitica/const.py

```diff
@@ -26,3 +26,5 @@
 ATTR_SKILL = "skill"
 ATTR_TASK = "task"
 SERVICE_CAST_SKILL = "cast_skill"
+
+DEVELOPER_ID = "4c4ca53f-c059-4ffa-966e-9d29dd405daf"
```

Comment:
Whose dev id is this?

Reply:
That's my User ID. As I'm currently the most active in maintaining the integration, I thought that, in case Habitica devs need to contact someone, providing my user ID would be the most obvious choice.